lib/access_check_filter.js

```diff
@@ -4,6 +4,35 @@ const mwUtil = require('./mwUtil');
 const P = require('bluebird');
 const URI = require('hyperswitch').URI;
 
+function redirect(content, location, options) {
+    const vary = content.headers.vary ? `${content.headers.vary}, origin` : 'origin';
+    return {
+        status: 302,
+        headers: Object.assign(content.headers, {
+            location,
+            'cache-control': options.redirect_cache_control || 'no-cache',
+            vary
+        }),
+        body: content.body
+    };
+}
+
+function response(hyper, req, path, titleParamName, location) {
+    const pathBeforeTitle = path.substring(0, path.indexOf(`{${titleParamName}}`));
+    const contentLocation = new URI(pathBeforeTitle, req.params, true).toString() + location;
+    return hyper.request({
+        method: req.method,
+        uri: new URI(contentLocation),
```

Comment:
I see, good point. Since this is a more general testing issue, perhaps it would be worth adding a http filter rule that redirects requests to the REST API directly to RB itself when testing?